tex/frontmatter/digraph.tex

```diff
@@ -3,30 +3,31 @@
 
 \bgroup
 \renewcommand{\href}[1]{} % temp disable links
+\hypersetup{linkcolor=black} % temporarily make internal refs black, so part labels are not all maroon
```

Comment:
I actually think this is not super necessary (having it colored makes it a little more obvious the labels are clickable --- although I will probably add a one-sentence paragraph to explicitly say this right before the digraph itself).